### PR TITLE
Add instructions for JSBSim SITL simulation

### PR DESCRIPTION
**Pull request by Jaeyoung-Lim**
**Problem Description**
JSBSim SITL simulation has been added in https://github.com/PX4/Firmware/pull/15748 

This commit adds instructions for setting up/running SITL simulations for jsbsim

Sorry for being late on posting the instructions since this was added a while ago to firmware.

### CODE REVIEW

File: en/simulation/jsbsim.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+make px4_sitl jsbsim
+```
+If you want to run without the flightgear UI, you can add `HEADLESS=1` to the front of the make command.
+
+
+The supported vehicles and `make` commands are listed below (click on the links to see the vehicle images).
+
+Vehicle | Command
+--- | ---
+[Standard Plane]() | `make px4_sitl jsbsim_rascal`
```
Comment (hamishwillee) [inline en/simulation/jsbsim.md]: Missing links for these three

**Comment by hamishwillee** [comment]
@Jaeyoung-Lim Looks good. I have subedited (please check). There are some missing link targets.

Most important, you need to add a SUMMARY.md page target.

File: en/simulation/jsbsim.md (hunk 1 above)
Comment (Jaeyoung-Lim) [reply en/simulation/jsbsim.md]: Right, would it be possible to add this in the next PR?

**Comment by Jaeyoung-Lim** [comment]
@hamishwillee Thanks! I added the SUMMARY.md page target :smile:

File: en/simulation/jsbsim.md (hunk 1 above)
Comment (hamishwillee) [reply en/simulation/jsbsim.md]: Sure, but because I want to avoid the build warnings I have changed these to not be links.

**Comment by hamishwillee** [comment]
Awesome. Merging!

**Pull request merged**